boards/arm/nucleo_l412rb_p/nucleo_l412rb_p.dts

```diff
@@ -78,6 +78,19 @@
 	status = "okay";
 };
 
+&adc1 {
+	/* PA2 and PA3 are used by LPUART1 */
+	pinctrl-0 = <&adc1_in1_pc0 &adc1_in2_pc1
+		&adc1_in3_pc2 &adc1_in4_pc3
+		&adc1_in5_pa0 &adc1_in6_pa1
+		&adc1_in7_pa2 &adc1_in8_pa3
+		&adc1_in9_pa4 &adc1_in10_pa5
+		&adc1_in11_pa6 &adc1_in12_pa7
+		&adc1_in13_pc4 &adc1_in14_pc5
+		&adc1_in15_pb0 &adc1_in16_pb1>;
```

Comment:
I'm not so sure about the usefulness of adding so many adc pins.
I think it would be better to just add ~1-3, ideally at header connections usually used for adcs and w/o other conflicts.
In case anyone needs more, they can easily be added using an overlay file.

Reply:
> I'm not so sure about the usefulness of adding so many adc pins.
> I think it would be better to just add ~1-3, ideally at header connections usually used for adcs and w/o other conflicts.
> In case anyone needs more, they can easily be added using an overlay file.

That is something I thought about but wasn't sure. I'm guessing potential pin conflicts are the issue or?

Reply:
I removed all but one of the channels and used Arduino pin A0 similar to Nucleo G474RE board.